### ISSUE TITLE
Property escape sequences false positive

### ISSUE THREAD

**Issue by ccleve** [AzureMarker/intellij-lalrpop#52]
```
grammar;
pub Term = { <Word> };
Word: String = <w:r"\p{L}"> => w;

```
There's a red squiggle under `\p{L]}` that says "Property escape sequences are not supported in this regex dialect", but the code compiles and runs fine.

**Comment by AzureMarker** [comment]
Looks like the plugin is using a different regex dialect than what LALRPOP accepts.

**Comment by AzureMarker** [comment]
The fix is releasing with 0.2.9
